wrappers/csharp/Intel.RealSense/Intel.RealSense.csproj

```diff
@@ -15,44 +15,65 @@
     <PackageTags>3d;vision;robotics;depth;realsense;perception</PackageTags>
     <PackageLicenseUrl>https://github.com/IntelRealSense/librealsense/blob/master/LICENSE</PackageLicenseUrl>
     <Configurations>Debug;Release</Configurations>
+    <Platforms>AnyCPU;x86;x64</Platforms>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include=".nuget\Intel.RealSense.targets" />
+    <Compile Remove="Properties/**" />
+  </ItemGroup>
+
   <PropertyGroup>
     <BuildPlatform>$(Platform)</BuildPlatform>
-    <BuildPlatform Condition="'$(BuildPlatform)' == 'AnyCPU'">x86</BuildPlatform>
+    <BuildPlatform Condition="'$(Platform)'=='AnyCPU'">x86</BuildPlatform>
+    <BuildPlatform Condition="'$(Platform)'=='AnyCPU' and '$(PROCESSOR_ARCHITEW6432)'=='AMD64'">x64</BuildPlatform>
     <InstallPath>C:\Program Files (x86)\Intel RealSense SDK 2.0\bin</InstallPath>
+    <BuildPathx86></BuildPathx86>
+    <BuildPathx64></BuildPathx64>
     <BuildPath Condition="'$(BuildPath)'==''">..\..\..\build\$(Configuration)\realsense2.dll</BuildPath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Include=".nuget\Intel.RealSense.targets" />
-    <Compile Remove="Properties/**" />
-  </ItemGroup>
-
-  <Target Name="CopyLocalDependencies" BeforeTargets="Build" Condition="'$(IsCrossTargetingBuild)' != 'true'">
-    <Copy Condition="Exists('$(BuildPath)')" SourceFiles="$(BuildPath)" DestinationFolder="$(OutputPath)" />
-    <Copy Condition="!Exists('$(BuildPath)') and Exists('$(InstallPath)')" SourceFiles="$(InstallPath)\$(BuildPlatform)\realsense2.dll" DestinationFolder="$(OutputPath)" />
+  <Target Name="IncludeLib" DependsOnTargets="IncludeDependencies" BeforeTargets="Build" Condition="'$(IsCrossTargetingBuild)' == 'true'">
+    <Copy Condition="'$(LibPathx86)'!=''" SourceFiles="$(LibPathx86)" DestinationFolder="$(OutputPath)\runtimes\win-x86\native" />
+    <Copy Condition="'$(LibPathx64)'!=''" SourceFiles="$(LibPathx64)" DestinationFolder="$(OutputPath)\runtimes\win-x64\native" />
+    <ItemGroup>
+      <Frameworks Include="$(TargetFrameworks)" />
+    </ItemGroup>
+    <Copy Condition="'$(BuildPlatform)'=='x86' and '$(LibPathx86)'!=''" SourceFiles="$(LibPathx86)" DestinationFolder="$(OutputPath)\%(Frameworks.Identity)" />
```

Comment:
What is the difference between `LibPathx86` and `LibPath86`?

Reply:
The LibPath[86/64] properties (without x) are used locally within the EnsureBitCorrectness target before being set to LibPathx[86/64] properties (with x) which are the "returned" values. I'm not sure how scoping works within targets of csproj files which is why I haven't named them identically.

Reply:
Latest commit moves all of these to LibPathx[86/64] properties for neatness